build/tools.mk

```diff
@@ -11,8 +11,26 @@ GO_MOD_OUTDATED  ?= go-mod-outdated
 TOOL_DIR     ?= tools
 TOOL_CONFIG  ?= $(TOOL_DIR)/tools.go
 
+TOOLS_IMAGE ?= zalegrala/tempo-ci-tools
```

Comment:
Once we have an image published, we can switch to using it.  This PR creates the image publication as well as the implementation.  We can break this into two PRs if desired, but I thought leaving it as one gave a more holistic picture of what is happening.

Reply:
yeah, let's clean it up in follow up...